src/main/resources/uk/ac/ebi/subs/data/submittable/seq_var_analysis_mapping.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xml-bindings xmlns="http://www.eclipse.org/eclipselink/xsds/persistence/oxm"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://www.eclipse.org/eclipselink/xsds/persistence/oxm http://www.eclipse.org/eclipselink/xsds/eclipselink_oxm_2_4.xsd"
+              package-name="uk.ac.ebi.subs.data.submittable">
+    <java-types>
+        <java-type name="Submittable" xml-transient="true"/>
+        <java-type name="BaseSubmittable" xml-transient="true"/>
+        <java-type name="AbstractENASubmittable" xml-transient="true"/>
+        <java-type name="Analysis" xml-transient="true"/>
+        <java-type name="ENASequenceVariationAnalysis" xml-accessor-type="NONE">
+            <xml-root-element name="ANALYSIS"/>
+            <xml-type prop-order="accession alias teamName title description studyRefs sampleRefs
+                assemblyName experimentType programs platforms imputation files enaAttributeList"/>
+            <java-attributes>
+                <xml-attribute java-attribute="accession"/>
+                <xml-attribute java-attribute="alias"/>
+                <xml-attribute java-attribute="teamName" name="center_name"/>
+                <xml-element java-attribute="accession" name="IDENTIFIERS/PRIMARY_ID"/>
+                <xml-element java-attribute="alias" xml-path="IDENTIFIERS/SUBMITTER_ID/text()"/>
+                <xml-attribute java-attribute="teamName" xml-path="IDENTIFIERS/SUBMITTER_ID/@namespace"/>
```

Comment:
This is the 2nd teamName attribute

Reply:
good catch, I copied that from the existing mappings. I've fixed it in the mappings for all submittables